source/app-analysis/run-analysis.txt

```diff
@@ -0,0 +1,84 @@
+.. _rm-run-app-analysis:
+
+==========================
+Run Pre-Migration Analysis
+==========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can run Pre-Migration Analysis against a relational database to
+review potential migration risks before migrating data to MongoDB. Running
+Pre-Migration Analysis generates a report that identifies risks and provides
+guidance on how to mitigate them.
+
+About this Task
+---------------
+
+Relational Migrator typically takes 5-10 minutes to crawl the database, though
+it can take longer if the machine running Relational Migrator isn't near the
+database server.
+
+.. important::
+
+   Relational Migrator supports Pre-Migration Analysis for Microsoft SQL
+   Server, MySQL, Oracle, and PostgreSQL. Pre-Migration Analysis isn't
+   supported for databases connected through a JDBC driver. For more
+   information on supported databases and versions, see 
+   :ref:`supported-databases`.
+
+Before you Begin
+----------------
+
+To run Pre-Migration Analysis, you need a :ref:`project with a live database
+connection <rm-create-project-live>`.
+
+Steps
+-----
+
+.. procedure::
+   :style:  normal
+
+   .. step:: Open your project
+
+      If your project has no Pre-Migration Analysis reports, Relational
+      Migrator displays help text to suggest analyzing your application.
+
+   .. step:: Navigate to the :guilabel:`Pre-Migration Analysis` tab
+
+      You can review previous reports here, or run Pre-Migration Analysis.
+
+   .. step:: Select a relational database connection if one isn't already set
+      
+      In the Database Connections bar, select the saved connection from the 
+      :guilabel:`Relational DB` drop-down.
```

Comment:
[nit]

I think this should be "dropdown" here as I think we're using the noun form.